src/test/java/com/redhat/red/build/koji/HttpClientUtilsTest.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (C) 2017 Red Hat, Inc. (jcasey@redhat.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.red.build.koji;
+
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.io.InputStream;
+
+import static org.apache.http.client.utils.HttpClientUtils.closeQuietly;
+import static org.mockito.Mockito.never;
+
+public class HttpClientUtilsTest
+{
+
+    @Test
```

Comment:
Can you add some short docs on what each of these tests is trying to accomplish?